peps/pep-0752.rst

```diff
@@ -133,8 +133,9 @@ Parent Namespace
     A namespace's parent refers to the namespace without the trailing
     hyphenated component e.g. the parent of ``foo-bar`` is ``foo``.
 Child Namespace
-    A namespace's child refers to the namespace with an additional trailing
-    hyphenated component e.g. the child of ``foo`` is ``foo-bar``.
+    A namespace's child refers to the namespace with additional trailing
+    hyphenated components e.g. ``foo-bar`` is a valid child of ``foo`` as is
+    ``foo-bar-baz``.
```

Comment:
To match the API:

```rst
* ``children``: This is an array of any child namespaces. For example, if the
  namespace is ``foo`` and there are active grants for ``foo-bar`` and
  ``foo-bar-baz`` then this would be ``["foo-bar", "foo-bar-baz"]``.
```